CODEOWNERS

```diff
@@ -121,7 +121,7 @@ Kconfig*                                  @tejlmand
 /subsys/esb/                              @Raane @lemrey
 /subsys/event_manager/                    @pdunaj
 /subsys/fw_info/                          @hakonfam
-/subsys/mpsl/                             @joerchan @hurricaneFromMoscow
+/subsys/mpsl/                             @joerchan @carlopaparo
```

Comment:
Looks like we are missing an entry fro /inlude/mpsl

Reply:
Added you and @carlopaparo